fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/matchers/PlannerBindings.java

```diff
@@ -32,10 +31,10 @@
 
 /**
  * A map-like structure that supports a map from a binding to a collection of {@link Bindable}s, such as
- * {@link PlannerExpression}s and {@link ExpressionRef}s. A binding's key is a pointer to the {@link ExpressionMatcher}
- * that created the binding, eliminating the need for a unique string or symbol identifier. A {@code PlannerBindings}
- * is immutable but has a {@link Builder} that can be used to build up a set of bindings incrementally. Additionally,
- * bindings can be combined using {@link #mergedWith(PlannerBindings)}.
+ * {@link com.apple.foundationdb.record.query.plan.temp.RelationalExpression}s and {@link ExpressionRef}s. A binding's
```

Comment:
use unqualified names if possible

Reply:
Our style checker requires us to use qualified names here because `RelationalExpression` is in a different package. If we import it but only use it in a Javadoc, the unused import linter rule gets upset.

I'd love for this to be fixed, but I wasn't successful last time I tried (some years ago).